cannon/mipsevm/tests/difftester.go

```diff
@@ -125,43 +129,72 @@ func (d *DiffTester[T]) run(t testRunner, testCases []T, opts ...TestOption) {
 	cfg := newTestConfig(t, opts...)
 	for _, vm := range cfg.vms {
 		for i, testCase := range testCases {
-			randSeed := randomSeed(t, d.testNamer(testCase), i)
+			randSeed := cfg.randomSeed
+			if randSeed == 0 {
+				randSeed = randomSeed(t, d.testNamer(testCase), i)
+			}
```

Comment:
Made the random seed configurable, so we can inject it for fuzz tests.  Otherwise, we generate the seed.